deps/rabbit/src/rabbit_fifo.erl

```diff
@@ -1500,14 +1521,30 @@ return_all(Meta, #?MODULE{consumers = Cons} = State0, Effects0, ConsumerId,
                 end, {State, Effects0}, Checked).
 
 %% checkout new messages to consumers
-checkout(#{index := Index} = Meta, OldState, State0, Effects0) ->
+checkout(Meta, OldState, State, Effects) ->
+    checkout(Meta, OldState, State, Effects, true).
```

Comment:
Would it be better to default to false here in terms of number of lines changes?

Reply:
Not really, there are 2 more 'true' than 'false'